fhir-swagger-generator/pom.xml

```diff
@@ -11,6 +11,12 @@
     <artifactId>fhir-swagger-generator</artifactId>
 
     <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>fhir-examples</artifactId>
+            <version>${project.version}</version>
+            <scope>compile</scope>
```

Comment:
these are not picked up in any other project? if so, we may want to add an exclusion. 

Double checking with this comment.

Reply:
Nothing else should depend on this one...its basically an independent project for making some swagger / openapi definitions (similar to fhir-tools, except that it depends on fhir-model).